app/views/users/sessions/new.html.slim

```diff
@@ -0,0 +1,16 @@
+= provide :title, 'Sign In'
+section.main-content.container
```

Comment:
I could not find a way to do this that did not require making all the static content into partials.  That seemed like an non-intuitive thing to do in order to abstract out some common formatting.